homeassistant/components/netatmo/sensor.py

```diff
@@ -282,42 +282,27 @@ def __init__(self, data_handler, data_class_name, module_info, sensor_type):
                 f"{module_info.get('module_name', device['type'])}"
             )
 
-        self._name = (
+        self._attr_name = (
             f"{MANUFACTURER} {self._device_name} {SENSOR_TYPES[sensor_type][0]}"
         )
         self.type = sensor_type
-        self._state = None
+        self._attr_state = None
         self._device_class = SENSOR_TYPES[self.type][3]
-        self._icon = SENSOR_TYPES[self.type][2]
-        self._unit_of_measurement = SENSOR_TYPES[self.type][1]
+        self._attr_icon = SENSOR_TYPES[self.type][2]
+        self._attr_unit_of_measurement = SENSOR_TYPES[self.type][1]
         self._model = device["type"]
-        self._unique_id = f"{self._id}-{self.type}"
+        self._attr_unique_id = f"{self._id}-{self.type}"
         self._enabled_default = SENSOR_TYPES[self.type][4]
 
-    @property
-    def icon(self):
-        """Icon to use in the frontend, if any."""
-        return self._icon
-
     @property
     def device_class(self):
         """Return the device class of the sensor."""
         return self._device_class
 
-    @property
-    def state(self):
-        """Return the state of the device."""
-        return self._state
-
-    @property
-    def unit_of_measurement(self):
-        """Return the unit of measurement of this entity, if any."""
-        return self._unit_of_measurement
-
     @property
     def available(self):
         """Return entity availability."""
-        return self._state is not None
+        return self._attr_state is not None
 
     @property
     def entity_registry_enabled_default(self) -> bool:
```

Comment:
Can't place a suggestion on this part for some reason, but this property can be removed.